### DISCUSSION TITLE
post/nju-ics-pa-season1/

### DISCUSSION THREAD

**Discussion by giscus** [lyc8503/lyc8503#33] [Announcements]
# post/nju-ics-pa-season1/

在南大计算机系的大一新生之中一直流传着一个神秘的传说, 它的名字叫 ICS PA. 其实是大二计算机系统基础课程的一份难度相对较大但设计的很好的课程大作业. 目标是实现一个”青春版” QEMU – Nemu, 实现 ALU, FPU, x86 指令集, 保护模式, 内存 cache / 分页 / 分段, 外设 IO 功能的虚拟机, 并运行一些简单的程序 (简化了很多实现, 现代

https://blog.lyc8503.net/post/nju-ics-pa-season1/

**Comment by xcbbbb** [comment]
请问下用wsl+Ubuntu能做吗

**Reply by lyc8503** [reply]
我做的 2020 年版本的, 你参考新版的系统要求, 符合的话, WSL 本身带来的问题应该不大.
